stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -3531,6 +3566,22 @@ impl NakamotoChainState {
 
             marf_keys.push(nakamoto_keys::ongoing_tenure_id().to_string());
             marf_values.push(nakamoto_keys::make_tenure_id_value(&tenure_id));
+        } else {
+            // if we are here (no new tenure or tenure_extend) we need to accumulate the parent total tenure size
+            if let Some(current_total_tenure_size) =
```

Comment:
When would this ever be `None`?  The default value for the tenure size is 0 from the migration, and this is always set, right?

Reply:
Correct, do you want me to explicitely manage it?